src/serizlizer/serializable.cr

```diff
@@ -9,7 +9,7 @@ module Serializer
       # Returns json root key.
       #
       # Default data root key is `"data"`. This behavior can be override by overriding this method.
-      #
+      # It can be omited be setting nil, but any meta-options sent will be ignored.
```

Comment:
"be setting" -> "by setting". Also please mention that any `meta` data in this case will be omitted as well